src/lr_apply.rs

```diff
@@ -1,6 +1,333 @@
+use crate::include::common::bitdepth::BitDepth;
+use crate::include::dav1d::headers::Rav1dPixelLayout;
+use crate::include::dav1d::headers::RAV1D_RESTORATION_NONE;
+
+use crate::include::dav1d::headers::RAV1D_RESTORATION_SGRPROJ;
+use crate::include::dav1d::headers::RAV1D_RESTORATION_WIENER;
+
+use crate::src::align::Align16;
+use crate::src::internal::Rav1dDSPContext;
+use crate::src::internal::Rav1dFrameContext;
+use crate::src::lf_mask::Av1RestorationUnit;
+use crate::src::looprestoration::looprestorationfilter_fn;
+use crate::src::looprestoration::LooprestorationParams;
+use crate::src::looprestoration::LrEdgeFlags;
+use crate::src::looprestoration::LR_HAVE_BOTTOM;
+use crate::src::looprestoration::LR_HAVE_LEFT;
+use crate::src::looprestoration::LR_HAVE_RIGHT;
+use crate::src::looprestoration::LR_HAVE_TOP;
+use crate::src::tables::dav1d_sgr_params;
+
+use libc::c_void;
+use libc::memcpy;
+use libc::ptrdiff_t;
+use std::cmp;
+use std::ffi::c_int;
 use std::ffi::c_uint;
 
 pub type LrRestorePlanes = c_uint;
 pub const LR_RESTORE_V: LrRestorePlanes = 4;
 pub const LR_RESTORE_U: LrRestorePlanes = 2;
 pub const LR_RESTORE_Y: LrRestorePlanes = 1;
+
+// TODO(perl) Temporarily pub until mod is deduplicated
+pub(crate) unsafe fn lr_stripe<BD: BitDepth>(
```

Comment:
Like #586, we shouldn't need these temporary `pub(crate)`s anymore since now the module is deduplicated.

Reply:
right, thanks